posthog/warehouse/api/external_data_schema.py

```diff
@@ -369,7 +369,14 @@ def incremental_fields(self, request: Request, *args: Any, **kwargs: Any):
             ]
         elif source.source_type == ExternalDataSource.Type.DOIT:
             incremental_columns = DOIT_INCREMENTAL_FIELDS
-
+        # TODO: Incremental syncing
+        # elif source.source_type == ExternalDataSource.Type.GOOGLESHEETS:
+        #     sf_schemas = get_snowflake_schemas(SnowflakeSourceConfig.from_dict(source.job_inputs))
+        #     columns = sf_schemas.get(instance.name, [])
+        #     incremental_columns = [
+        #         {"field": name, "field_type": field_type, "label": name, "type": field_type}
+        #         for name, field_type in filter_snowflake_incremental_fields(columns)
+        #     ]
```

Comment:
Seems off to have some of the snowflake code commented here. I suppose incremental syncing would be disabled for this.

Reply:
Oops, forgot to push my last commit - sorted now